Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,886 @@
+        3. Set the previous TLS versions on all SQL Servers in the Subscription.
+# Instructions to execute the script:
+    To remediate:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to set the required TLS version in all SQL Servers in the Subscription. Refer `Examples`, below.
+    To roll back:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to set the previous TLS versions in all SQL Servers in the Subscription. Refer `Examples`, below.
+# Examples:
+    To remediate:
+        1. To review the SQL Servers in a Subscription that will be remediated:
+           Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -DryRun
+        2. To set minimal required TLS version  of all SQL Servers in a Subscription:
+           Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck
+        3. To set minimal required TLS version on the of all SQL Servers in a Subscription, from a previously taken snapshot:
+           Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\setMinTLSVersionForSQLServers\SQLServersWithoutMinReqTLSVersion.csv
+        4. To set minimal required TLS version of all SQL Servers in a Subscription without taking back up before actual remediation:
+           Set-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -SkipBackup
+        To know more about the options supported by the remediation command, execute:
+        Get-Help Set-SQLServerRequiredTLSVersion -Detailed
+    To roll back:
+        1. To reset minimal required TLS version of all SQL Servers in a Subscription, from a previously taken snapshot:
+           Reset-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\setMinTLSVersionForSQLServers\RemediatedSQLServers.csv
+        
+        2. To reset minimal required TLS version of all SQL Servers in a Subscription, from a previously taken snapshot:
+           Reset-SQLServerRequiredTLSVersion -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\setMinTLSVersionForSQLServers\RemediatedSQLServers.csv
+        To know more about the options supported by the roll back command, execute:
+        Get-Help Reset-SQLServerRequiredTLSVersion -Detailed        
+###>
+
+
+function Setup-Prerequisites
```

Comment:
Kindly refer to this Script - https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1 - and make the changes accordingly.

Reply:
Updated BRS according to other BRSs

Reply:
It is not according to the standard, kindly refer the script which I provided.

Reply:
Followed the script provided to following all standard code